docs/internal/other_modules.rst

```diff
@@ -0,0 +1,44 @@
+
+Other Modules
+=============
+
+To use these modules, you will need the public modules available:
+
+    module use /g/data/v10/public/modules/modulefiles
+
+Pycharm
+-------
+
+Pycharm (Community Edition) is a free
+The easiest way to run Pycharm is to copy the launcher on to your desktop.
+
+You can do this by running:
+
+    cp /g/data/v10/public/modules/pycharm/PyCharm.desktop ~/Desktop/
+
+
+To upgrade the version, download the latest tarball from the jetbrains website and place it in the folder
+``/g/data/v10/public/modules/pycharm/``
```

Comment:
It'd be good to mark this as **for administrators only**, or move it somewhere else.

I think splitting documentation by intended user and task will make it more useful, rather than by software module.

Reply:
Since it is only one line, I'll make it as admin only. If it gets any bigger, I'll move it to the internal section.